libraries/AP_AHRS/AP_AHRS_View.h

```diff
@@ -175,7 +174,7 @@ class AP_AHRS_View
 
     // Logging Functions
     void Write_AttitudeView(const Vector3f &targets) const;    
-    void Write_Rate( const AP_Motors &motors, const AC_AttitudeControl &attitude_control,
```

Comment:
why do we need the addition of "class"?

Reply:
I also vote for removing that little space after the bracket.. but obviously not a blocker.

Reply:
> I also vote for removing that little space after the bracket.. but obviously not a blocker.

Done.  It was bugging me, too.